types/storybook__addon-knobs/tsconfig.json

```diff
@@ -27,7 +27,10 @@
         "forceConsistentCasingInFileNames": true
     },
     "files": [
+        "angular.d.ts",
         "index.d.ts",
-        "storybook__addon-knobs-tests.tsx"
+        "react.d.ts",
+        "storybook__addon-knobs-tests.tsx",
+        "bue.d.ts"
```

Comment:
`bue` -> `vue`

Reply:
🤦‍♂️ fixed in ce620ace2855b7e4a82bf818aa9890a547d46bcc